docs/UserGuide.md

```diff
@@ -3,71 +3,43 @@ layout: page
 
 ![Ui](images/icon.png){:style="display:block; margin-left:auto; margin-right:auto"}
 
 
-Now you will never miss any internship application deadlines or interviews again!
+Now you will never miss any application deadlines or internship interviews again.
 PleaseHireUs has been optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, PleaseHireUs can get your internship management tasks done faster than traditional GUI apps.
-We hope you find PleaseHireUs to be very useful in your internship hunt!
+We hope you find PleaseHireUs to be very useful in your internship hunt.
 
 <div style="page-break-after: always;"></div>
 
+## Table of Content
 * Table of Contents
 {:toc}
 <div style="page-break-after: always;"></div>
 
-## Quick start
-
-1. Ensure you have Java `11` or above installed in your Computer. You can refer to [this](#appendix-installing-java-11)
-
-2. Download the latest `PleaseHireUs.jar` from [here](https://github.com/AY2223S1-CS2103T-W17-4/tp/releases).
-
-3. Copy the file to the folder you want to use as the _home folder_ for your application.
-
-4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
-
-5. Type the command in the command box and press <button>enter</button> to execute it. e.g. typing **`help`** and pressing <button>enter</button> will open the help window.<br>
-   
-6. Some example commands you can try you may refer to [this](#features) for details of each command.
-
-* **`list`** : Lists all internships.
-
-* **`add n/Grab p/software engineer pr/ASSESSMENT web/https://www.grab.com/sg/about`** : Adds a new internship to the internship tracker.
-
-* **`delete 3`** : Deletes the 3rd internship shown in the current list.
-
-* **`clear`** : Deletes all internships.
-
-* **`exit`** : Exits the app. 
-
-<div style="page-break-after: always;"></div>
-
 ## About this User Guide
 
 This guides aims to: <br>
 
 1. Teach first-time users how to start using PleaseHireUs
-2. Show and explain to users how to use each command
-3. Provide users with a summary of the commands available
+2. Show and explain to users how to use the features available
+3. Provide users with a summary of the features available
 4. Provide advance users with tips to improve their experience
 
-
 ## Navigating the User Guide 
 **Information Box**
 <div markdown="block" class="alert alert-info">
-**:information_source: Info:** Provides extra information that is useful
+**:information_source: Info:** Provides information that is useful
 </div>
 
 **Tip Box**
 <div markdown="block" class="alert alert-success">
-**:bulb: Tip:** Provides pointers to enhance your experience using the application 
+**:bulb: Tip:** Provides advance users with pointers to enhance your experience
```

Comment:
I think should be "to enhance their experience"

Reply:
updated